roles/pve_exporter/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+- name: Preflight
+  ansible.builtin.include_tasks:
+    file: preflight.yml
+  tags:
+    - pve_exporter_install
+    - pve_exporter_configure
+    - pve_exporter_run
+
+- name: Credentials
+  ansible.builtin.import_tasks: credentials.yml
+  when: pve_exporter_set_credential | bool
+
+- name: Install
+  tags:
+    - pve_exporter_install
+  block:
+    - name: Install packages dependencies
+      ansible.builtin.package:
+        name:
+          - python3
+          - python3-pip
+          - python3-virtualenv
+        state: present
+    - name: "Create system group {{ pve_exporter_system_group }}"
+      ansible.builtin.group:
+        name: "{{ pve_exporter_system_group }}"
+        state: present
+    - name: "Create system user {{ pve_exporter_system_user }}"
+      ansible.builtin.user:
+        name: "{{ pve_exporter_system_user }}"
+        system: true
+        shell: "/usr/sbin/nologin"
+        group: "{{ pve_exporter_system_group }}"
+        home: "{{ pve_exporter_home }}"
+        create_home: true
+      become: true
```

Comment:
Please use the logic for creating the users from the `_common` role, where users are not modified if they do not exist, this is to prevent making changes to system users unexpectedly.